includes/Products/Feed.php

```diff
@@ -62,6 +67,9 @@ private function add_hooks() {
 
 		// handle the feed data request
 		add_action( 'woocommerce_api_' . self::REQUEST_FEED_ACTION, array( $this, 'handle_feed_data_request' ) );
+
+		// Send request fir feed one time upload after feed file generated
+ 		add_action( 'wc_facebook_feed_generation_completed', array( $this, 'send_request_to_upload_feed' ) );
 	}
 
 
```

Comment:
// fpassthru might be disabled in some hosts (like Flywheel)
^ what do we do if fpassthru is allowed? We don't write anything in this case as far as I can tell @mshymon